source/include/azure_iot_hub_client.h

```diff
@@ -45,11 +45,11 @@ typedef enum AzureIoTHubMessageQoS
  */
 typedef enum AzureIoTHubMessageType
 {
-    eAzureIoTHubCloudToDeviceMessage = 1,    /**< The message is a cloud message. */
-    eAzureIoTHubDirectMethodMessage,         /**< The message is a direct method message. */
-    eAzureIoTHubTwinGetMessage,              /**< The message is a twin get response (payload contains the twin document). */
-    eAzureIoTHubTwinReportedResponseMessage, /**< The message is a twin reported property status response. */
-    eAzureIoTHubTwinDesiredPropertyMessage,  /**< The message is a twin desired property message (incoming from the service). */
+    eAzureIoTHubCloudToDeviceMessage = 1,          /**< The message is a cloud message. */
+    eAzureIoTHubCommandMessage,                    /**< The message is a command message. */
+    eAzureIoTHubPropertiesGetMessage,              /**< The message is a property get response (payload contains the property document). */
+    eAzureIoTHubPropertiesReportedResponseMessage, /**< The message is a reported property status response. */
+    eAzureIoTHubPropertiesDesiredPropertyMessage,  /**< The message is a desired property message (incoming from the service). */
```

Comment:
I think this should be 'eAzureIoTHubPropertiesWriteablePropertyMessage' and "This message is a writeable property message" or "tis mesage contains writeable properties" (though it breaks symmetry with other comments so maybe not as good).

Writeable is what DTDLv2 refers to this as.  

And I just popped open embedded SDK and i'm not doing anyone favors there, since I have `AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_DESIRED_PROPERTIES` but it should be `AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_WRITEABLE_PROPERTIES`.  I'll deal with that.

I think reported and get are OK for other names though..